repo/packages/N/neo4j-proxy/0/config.json

```diff
@@ -0,0 +1,28 @@
+{
+  "type": "object",
+  "properties": {
+    "neo4j-proxy": {
+      "description": "Neo4j specific configuration properties",
+      "type": "object",
+      "properties": {
+        "auth-username": {
+          "description": "Username for authentication",
+          "type": "string",
+          "default": "neo4j"
+        },
+        "auth-password": {
+          "description": "Password for authentication",
+          "type": "string",
+          "default": "dcos"
```

Comment:
the proxy UI says the default username/password is neo4j/neo4j. which one is correct?